client/my-sites/exporter/index.jsx

```diff
@@ -0,0 +1,26 @@
+/**
+ * External dependencies
+ */
+import React, { PropTypes, Component } from 'react';
+import { connect } from 'react-redux';
+
+/**
+ * Internal dependencies
+ */
+import Exporter from './exporter';
+import { toggleAdvancedSettings, toggleSection } from 'lib/site-settings/exporter/actions';
+
+function mapStateToProps( state ) {
+	return {
+		advancedSettings: state.siteSettings.exporter.ui.toJS().advancedSettings,
+	}
+}
+
+function mapDispatchToProps( dispatch ) {
+	return {
+		toggleAdvancedSettings: () => dispatch( toggleAdvancedSettings() ),
+		toggleSection: ( section ) => dispatch( toggleSection( section ) )
```

Comment:
Not sure how you would feel about doing this, but here is a great place to consider using functional composition. There's built-in support for this in **lodash**.

```js
import compose from 'lodash/function/compose';

toggleAdvancedSettings: compose( dispatch, toggleAdvancedSettings ),
toggleSection: compose( dispatch, toggleSection )
```

Now if either of these gets called, it will call the function on the right, then spit the output of those functions into the function on the left. Any arguments added to the call, such as `section` will apply to the function on the right.